src/utils/Xhr.js

```diff
@@ -23,6 +25,11 @@ type PayloadType = StringAnyMap | Array<StringAnyMap>;
 
 const DEFAULT_UPLOAD_TIMEOUT_MS = 120000;
 const MAX_NUM_RETRIES = 3;
+const DEFAULT_COMMON_HEADERS = {
+    Accept: 'application/json',
+    [HEADER_ACCEPT_LANGUAGE]: DEFAULT_LOCALE,
```

Comment:
This seems like a major change in disguise, as I believe the browser sets this for the user currently. Does this cause issues for other elements that aren't passing in the user-defined locale?

Reply:
In most cases with our APIs that look at accept-language, if it's missing it defaults to en-US.

I'm going to remove this. Thanks for catching it!